napari/utils/events/containers/_evented_list.py

```diff
@@ -71,6 +71,8 @@ class EventedList(TypedMutableSequence[_T]):
         emitted when ``index`` is set from ``old_value`` to ``value``
     reordered (value: self)
         emitted when the list is reordered (eg. moved/reversed).
+    duplicated (index: int, old_value: T, value: T)
+        emitted after the duplicate ``value`` of ``old_value`` was created and inserted at ``index``.
```

Comment:
did I correctly understand that inserted event still will be emitted? If yes, then this should be mentioned in the documentation.

Reply:
The inserted event is emitted when the `ll.insert` is called in the `_duplicate_layer` function, just like before since it is an insert. I will try to optimise the documentation :)

Reply:
I added a sentence about the insert event still being emitted before the duplicated event.